coala_langserver/jsonrpc/dispatchers.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Palantir Technologies, Inc.
```

Comment:
I would prefer that we did not copy other code into the repository.  For example, I can not easily check to see if you modified a line from the origin.

While there are some additional dependencies at https://github.com/palantir/python-language-server/blob/develop/setup.py#L34 , could we not require that it is installed, and then use the jsonrpc they provide?  If that is not possible, maybe we can coordinate with @palantir to split that out into a sharable library.

This tool is still a prototype, so it can have additional install steps , and keeping the origin of source code clear.

It also means that additional tests for the jsonrpc would be contributed to https://github.com/palantir/python-language-server , so everyone benefits.

Reply:
I will follow your recommendations, python-lang-server will be added as a dependency.